lib/lapis/error_codes.rb

```diff
@@ -17,6 +17,7 @@ class ErrorCodes
     ARCHIVER_FAILURE = 24
     ARCHIVER_ERROR = 25
     ARCHIVER_MISSING_KEY = 26
+    ARCHIVER_RATE_LIMITED = 30
```

Comment:
Why 30? Following the order, the next one seems to be 28?

Reply:
Thank you, fixed. I had got an error when using 28 before, but it seems to be working ok now.